distribution/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+set -o errexit
+
+# Prevent core dumps
+ulimit -c 0
+
+case $1 in -*) set -- vector "$@"; esac
```

Comment:
👀 

A comment might help here. I can't tell what this is supposed to be doing 😨 .

Reply:
Yep - all of these will have comments 👍